lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/MaplibreMap.kt

```diff
@@ -41,47 +40,47 @@ import kotlin.math.roundToInt
  * @param onMapLongClick Invoked when the map is long-clicked. See [onMapClick].
  * @param onFrame Invoked on every rendered frame.
  * @param isDebugEnabled Whether the map debug information is shown.
- * @param maximumFps The maximum frame rate at which the map view is rendered, but it can't excess
+ * @param maximumFps The maximum frame rate at which the map view is rendered, but it can't exceed
  *   the ability of device hardware.
  * @param logger kermit logger to use
- * @param overlay The composable content shown on top of the map.
- * @param content The map content additional to what is already part of the map as defined in the
- *   base map style linked in [styleUri].
+ * @param mapContent The map content additional to what is already part of the map as defined in the
+ * @param content The composable content shown on top of the map. base map style linked in
+ *   [styleUri].
```

Comment:
Whoops, this got jumbled up.

"base map style linked in [styleUri]" belongs to the documentation for `mapContent`.